viewer/packages/cad-model/src/batching/MultiBufferBatchingManager.ts

```diff
@@ -276,17 +287,19 @@ export class MultiBufferBatchingManager implements DrawCallBatchingManager {
     const newOffset = updateRange.byteOffset / typeSize;
     const newCount = updateRange.byteCount / typeSize;
 
-    const { offset: oldOffset, count: oldCount } = attribute.data.updateRange;
-    if (oldCount === -1) {
-      attribute.data.updateRange = { offset: newOffset, count: newCount };
+    if (attribute.data.updateRanges.length === 0) {
+      attribute.data.clearUpdateRanges();
+      attribute.data.updateRanges.push({ start: newOffset, count: newCount });
```

Comment:
Do we need to clear here? Isn't the updateRanges-array already empty?

Reply:
Guess we don't was just scared if there is some other metadata or something that it tracks internally.